CHANGELOG.md

```diff
@@ -2,49 +2,60 @@
 
 ## [Unreleased]
 
-- The version key in all configuration files (`buf.yaml`, `buf.gen.yaml`, `buf.work.yaml`) is now required.
-- The `--help` flag now writes to stdout instead of stderr.
-- Removed the `--source` flag on `buf build` in favor of the first positional parameter.
-- Removed the `--source-config` flag on `buf build` in favor of the `--config` flag.
-- Removed the `--file` flag on `buf build` in favor of the `--path` flag.
-- Removed the `--input` flag on `buf lint` in favor of the first positional parameter.
-- Removed the `--input-config` flag on `buf lint` in favor of the `--config` flag.
-- Removed the `--file` flag on `buf lint` in favor of the `--path` flag.
-- Removed the `--input` flag on `buf breaking` in favor of the first positional parameter.
-- Removed the `--input-config` flag on `buf breaking` in favor of the `--config` flag.
-- Removed the `--against-input` flag on `buf breaking` in favor of the `--against` flag.
-- Removed the `--against-input-config` flag on `buf breaking` in favor of the `--against-config` flag.
-- Removed the `--file` flag on `buf breaking` in favor of the `--path` flag.
-- Removed the `--input` flag on `buf generate` in favor of the first positional parameter.
-- Removed the `--input-config` flag on `buf generate` in favor of the `--config` flag.
-- Removed the `--file` flag on `buf generate` in favor of the `--path` flag.
-- Removed the `--input` flag on `buf ls-files` in favor of the first positional parameter.
-- Removed the `--input-config` flag on `buf ls-files` in favor of the `--config` flag.
-- Removed the `--file` flag on `buf export` in favor of the `--path` flag.
-- Removed the `--name` and `--dep` flags in `buf mod init`.
-- Removed the `--version` flag in favor of the `buf version` command, which writes to stdout.
-- Removed the `buf image build` command in favor of `buf build`.
-- Removed the `buf check breaking` command in favor of `buf breaking`.
-- Removed the `buf check lint` command in favor of `buf lint`.
-- Removed the `buf check ls-lint-checkers` command in favor of `buf config ls-lint-rules`.
-- Removed the `buf check ls-breaking-checkers` command in favor of `buf config ls-breaking-rules`.
-- Removed the `buf beta config init` command in favor of `buf config init`.
-- Removed the `buf beta mod export` command in favor of `buf export`.
-- Removed the `buf beta mod init` command in favor of `buf config init`.
-- Removed the `buf beta mod update` command in favor of `buf mod update`.
-- Removed the `buf beta mod clear-cache` command in favor of `buf mod clear-cache`.
-- Removed the `buf beta push` command in favor of `buf mod push`.
+This is our first v1.0 release candidate. This release largely concentrates on erroring for
+already-deprecated commands and flags.
+
+At Buf, we take compatibility very seriously. When we say v1.0, we mean it - we hope `buf` will be
+stable on v1 for the next decade, and if there is something we want to change, it is our responsibility to
```

Comment:
nit: We use `v1.0` and `v1` - we should probably just use one form. Maybe `v1` so that it suggests it includes all minors.

Reply:
Eh I thought about that but I kind of did this on purpose - v1.0 has more of a ring to it, but v1 is more correct in the stability context. Idk.

Reply:
Yea that's fair, feel free to leave as-is.